client/src/components/ProfileForm.js

```diff
@@ -23,26 +49,80 @@ const TableUserAttributes = ({ userAttributes }) => (
 			      hideOnScroll
     			/>
     		</Table.HeaderCell>
-				<Table.HeaderCell width={3}>Current Value</Table.HeaderCell>
-				<Table.HeaderCell width={5}>New Value</Table.HeaderCell>
+				<Table.HeaderCell width={3}>
+					Current Value
+				</Table.HeaderCell>
+				<Table.HeaderCell width={5}>
+					New Value
+				</Table.HeaderCell>
 			</Table.Row>
 		</Table.Header>
 		<Table.Body>
-			{userAttributes.map(userAttributes => {
-				const { name,value,type } = {...userAttributes}
+			{userAttributes.map((userAttributes, index) => {
+				const { name, type } = {...userAttributes}
+				let fieldName = []
+				if (type === 'string') {
+					fieldName = 'user.userAttributes[' + index + '].value'
+				}
 				return (
-					<Table.Row key={name}>
-						<Table.Cell>{name}</Table.Cell>	
-						{type === 'string' && <Table.Cell><div>{value} </div></Table.Cell>}
-						{type === 'string' && <Table.Cell><NewValue value={value} /></Table.Cell>}
+					<Table.Row key={index}>
+						<Table.Cell>
+							{name}
+						</Table.Cell>	
+						{type === 'string' && 
+						<Table.Cell>
+							{currValues[index].value} 
+						</Table.Cell>
+						}
+						{type === 'string' && 
+						<Table.Cell>
+							<Field
+								name={fieldName}
+								component={field => {
+														return (
+															<Form.Input
+																{...field.input}
+																type='string'
+																width={8}
+															/>	
+														)
+													}
+											}
+								type='string'									
+							/>
+						</Table.Cell>
+						}
+						{type === 'email' &&
+						<Table.Cell> 
+							{currValues[index].value}
+						</Table.Cell>
+						}
+						{type === 'email' &&
+						<Table.Cell>
+							<Field
+								name={fieldName}
+								component={field => {
+														return (
+															<Form.Input
+																{...field.input}
+																type='email'
+																width={8}
+															/>
```

Comment:
Do we need to show the initial value in the New Value field on the profile form?
Would it be better to have them edit each field individually rather than submitting the entire form?